types/map.go

```diff
@@ -228,3 +230,33 @@ func (m Map) IsNull() bool {
 func (m Map) IsUnknown() bool {
 	return m.Unknown
 }
+
+func (m Map) String() string {
+	if m.Unknown {
+		return attr.UnknownString
+	}
+
+	if m.Null {
+		return attr.NullString
+	}
+
+	// We want the output to be consistent, so we sort the output by key
+	keys := make([]string, 0, len(m.Elems))
+	for k := range m.Elems {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+
+	var res strings.Builder
+
+	res.WriteString("[")
```

Comment:
For a map, I'd suggest curly braces (e.g. `{`) to differentiate between collection types 👍

Reply:
I like it!